include/grpc/impl/codegen/port_platform.h

```diff
@@ -196,7 +196,7 @@
 #define GPR_GCC_TLS 1
 #define GPR_LINUX 1
 #define GPR_LINUX_LOG
-#define GPR_LINUX_MULTIPOLL_WITH_EPOLL 1
+#define GPR_POSIX_MULTIPOLL_WITH_POLL 1
```

Comment:
For testing you can use the environment variable 'GRPC_POLL_STRATEGY=poll' to force disabling epoll.